rotkehlchen/chain/ethereum/modules/nft/nfts.py

```diff
@@ -193,16 +193,13 @@ def get_db_nft_balances(self, filter_query: 'NFTFilterQuery') -> Dict[str, Any]:
             'total_usd_value': total_usd_value,
         }
 
-    def get_balances(
+    def fetch_nft_balances(
```

Comment:
```suggestion
    def query_balances(
```

nitpick: no need to mention nft, this is a method of the nft module. And query is probably the term you need, as you just query but not return.